superflore/generators/bitbake/run.py

```diff
@@ -76,7 +78,7 @@ def main():
         parser.error('Invalid args! --only requires specifying --ros-distro')
     if not selected_targets:
         selected_targets = get_distros_by_status('active')
-    repo_org = 'allenh1'
+    repo_org = 'ros'
```

Comment:
please change this to `lgvsl`, since we haven't come to a consensus about the `ros` org.

Reply:
We now have https://github.com/ros/meta-ros/ :-)

Reply:
Ah, okay. Nice!